docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2437,6 +2436,12 @@ urltransform = "/" regexp "/" replacement "/" modifiers
 
 In the `$urltransform` value, two characters must be escaped: the comma `,` and the dollar sign `$`. Use the backslash character `\` for this. For example, an escaped comma looks like this: `\,`.
 
+**Changing the origin**
```

Comment:
does it work in CoreLibs v1.15? if no, the compatibility section should be updated